webapp/components/Group/GroupForm.vue

```diff
@@ -123,6 +112,7 @@
         </ds-chip>
 
         <!-- location -->
+        <!-- TODO: move 'ds-select' from styleguide to main code and implement missing translation etc. functionality -->
```

Comment:
Why this comment, this is not needed? Does it help in any way?

Reply:
Is removed.